### PR TITLE
fix arraymap iterator remove issue

### PR DESCRIPTION
**Pull request by yantzu**
fix issue #370

### CODE REVIEW

**Comment by yantzu** [comment]
Travis failed seems due to https://github.com/travis-ci/travis-ci/issues/8199

File: google-http-client/src/test/java/com/google/api/client/util/ArrayMapTest.java [hunk 1]
```diff
@@ -100,4 +103,52 @@ public void testSet() {
...
+    ArrayMap<String, String> map = new ArrayMap<String, String>();
+    map.put("a", "a");
+    map.put("b", "b");
+    map.put("c", "c");
+    Iterator<Map.Entry<String,String>> iter = map.entrySet().iterator();
+    while (iter.hasNext()) {
+        Map.Entry<String,String> entry = iter.next();
+        if(!"all".equalsIgnoreCase(entry.getKey())){
+            iter.remove();
+        }
```
Comment (garrettjonesgoogle) [inline google-http-client/src/test/java/com/google/api/client/util/ArrayMapTest.java]: Use consistent indentation with the current code (2-space)

Comment (yantzu) [reply google-http-client/src/test/java/com/google/api/client/util/ArrayMapTest.java]: fixed. thanks!

**Comment by yantzu** [comment]
not sure what the googlebot want me to do. the code change committed by Matt is already in main git repository, right?

**Comment by lesv** [comment]
I'm going to close this as it appears to already be in the git repo.

I did verify that the CLA is correct.

**Comment by yantzu** [comment]
Hmmm..., I doesn't see my code in git repo dev branch

**Comment by lesv** [comment]
Yep.  Someone else is going to take over the maint of this repo when he returns next week.  I'll go over with him all the open PR's.

Apologies for the delay.

**Comment by chingor13** [comment]
The CLA was signed by the commit author.

**Pull request merged**